vms/platformvm/vm.go

```diff
@@ -259,6 +259,10 @@ func (vm *VM) Initialize(
 	return nil
 }
 
+func (vm *VM) WaitForEvent(ctx context.Context) (common.Message, error) {
+	return vm.Subscriber.WaitForEvent(ctx)
+}
```

Comment:
This is needed to disambiguate the multiple `WaitForEvent` embeddings. Will change this more in the followup platformvm PR.